### DISCUSSION TITLE
How to save column skipping callbacks and validations?

### DISCUSSION THREAD

**Discussion by vol1ura** [jeremyevans/sequel#1972] [General]
There is any solution to save column changes skipping callbacks and validations, like `update_column` in Rails does?

If I use `save(validate: false)` it skips validations, but can run callbacks.

**Comment by AlexWayfer** [comment]
https://sequel.jeremyevans.net/rdoc/files/doc/model_hooks_rdoc.html#label-Skipping+Hooks

Is this what do you want?

**Reply by vol1ura** [reply]
Thank you, yes, that's what I need
